content/modules/redisgears/installing-redisgears.md

```diff
@@ -0,0 +1,144 @@
+
+1. Install [Python 3](https://www.python.org/downloads/).
+1. To install the gears-cli, run: `pip install gears-cli`
+1. Download [rgsync offline package](https://docs.google.com/document/d/1xiNEi6O_BIHgyUDfhTqic0zD_DPJCSmIVzjWlCn-hCE/edit#heading=h.thlygbs3qwmo).
+1. Import the requirements:
+
+    ```src
+    # gears-cli import-requirements \
+        --host HOST [ --port PORT ] --password PASSWORD \
+        --requirements-path rgsync-99.99.99.linux-bionic-x64.zip
+    ```
+
+    {{< note >}}
+You can be more efficient and import only the requirements you need, but rgync is always required and can be combined with one or more of these packages according to your OS:
+
+- redisgears-requirement-v1-snowflake-sqlalchemy-linux-<os>-x64.zip
+- redisgears-requirement-v1-PyMySQL-linux-<os>-x64.zip
+- redisgears-requirement-v1-cx-Oracle-linux-<os>-x64.zip
+- redisgears-requirement-v1-cassandra-driver-linux-<os>-x64.zip
+
+This list can be different or more extensive in newer versions.
+    {{< /note >}}
+
+1. To verify the requirements were imported successfully, run: `RG.PYDUMPREQ`
+
+    This command returns a list of all available requirements.
+
+### Step 3: Register the Functions
+
+The following is a RedisGears recipe that shows how to use the Write Behind pattern to map data from Redis Hashes to MySQL tables.
+The recipe maps all Redis Hashes with the prefix `person:<id>` to the MySQL table persons, with `<id>` being the primary key and mapped to the person_id column.
+Similarly, it maps all Hashes with the prefix `car:<id>` to the cars table.
+
+```src
+from rgsync import RGWriteBehind
+from rgsync.Connectors import MySqlConnector, MySqlConnection
+
+'''
+Create MySQL connection object
+'''
+connection = MySqlConnection('demouser', 'Password123!', 'localhost:3306/test')
+
+'''
+Create MySQL persons connector
+persons - MySQL table to put the data
+person_id - primary key
+'''
+personConnector = MySqlConnector(connection, 'persons', 'person_id')
+
+personsMappings = {
+	'first_name':'first',
+	'last_name':'last',
+	'age':'age'
+}
+
+RGWriteBehind(GB, keysPrefix='person', mappings=personsMappings, connector=personsConnector, name='PersonsWriteBehind', version='99.99.99')
+
+'''
+Create MySQL car connector
+cars - MySQL table to put the data
+car_id - primary key
+'''
+carConnector = MySqlConnector(connection, 'cars', 'car_id')
+
+carsMappings = {
+	'id':'id',
+	'color':'color'
+}
+
+RGWriteBehind(GB, keysPrefix='cars', mappings=carsMappings, connector=carsConnector, name='CarsWriteBehind', version='99.99.99')
+```
+
+Go to the [rgsync website](https://pypi.org/project/rgsync/) to get the replication options and the configuration options for the database and mapping.
+
+1. Create a python file with the configuration mapping according to your specific needs.
+1. Create a file with the configuration mapping according to your specific needs: `gears-cli --host <host> --port <post> --password <password> <yourfile>.py`
```

Comment:
can this command be on a new line?  I'm a bit confused, sometimes commands are on new lines, sometimes inline

Reply:
Yeah. We're working on a convention. For now it's:

- short commands are inline
- long commands are on a new line

Reply:
ok